tests/test_tapioca.py

```diff
@@ -52,6 +52,28 @@ def test_in_operator(self):
         self.assertIn('other', response)
         self.assertNotIn('wat', response)
 
+    @responses.activate
+    def test_trasnform_camelCase_in_snake_case(self):
+        next_url = 'http://api.teste.com/next_batch'
+
+        responses.add(responses.GET, self.wrapper.test().data(),
+                      body='{"data": [{"key_snake": "value", "camelCase": "data in camel case"}], "paging": {"next": "%s"}}' % next_url,
+                      status=200,
+                      content_type='application/json')
+
+        responses.add(responses.GET, next_url,
+                      body='{"data": [{"key_snake": "value", "camelCase": "data in camel case"}], "paging": {"next": ""}}',
+                      status=200,
+                      content_type='application/json')
+
+        response = self.wrapper.test().get()
+
+
+        for item in response().pages():  
```

Comment:
It doesn't, but I was worried with this case too, since I change it in a recent past. Should I remove?